### DISCUSSION TITLE
Pickup Time only when Pickup is selected

### DISCUSSION THREAD

**Discussion by mergimrexha** [mattbanks/woocommerce-local-pickup-time#138] [Ideas]
is there an option to display a pickup time only if only on-site pickup has been selected?
Currently it is displayed all the time, whether it is a normal delivery or a pickup on site delivery

@timnolte

**Comment by knair97** [comment]
You could achieve this using CSS by making the entire pickup time component hidden when delivery is selected.

**Comment by tnchuntic** [comment]
This was mentioned in the plugin support section https://wordpress.org/support/topic/disable-the-pickup-time-for-other-shipping-methods/. I am not sure yet when the v1.4.x be released.

**Reply by timnolte** [reply]
The next release is close to being ready just working through some build issues related to dependency updates. My hope is to get the release out within the week.
